apps/dotcom/client/src/tla/components/dialogs/TlaSignInDialog.tsx

```diff
@@ -8,16 +9,28 @@ import {
 	TldrawUiDialogHeader,
 	TldrawUiDialogTitle,
 } from 'tldraw'
+import { routes } from '../../../routeDefs'
 import { defineMessages, F, useMsg } from '../../utils/i18n'
 import { TlaCtaButton } from '../TlaCtaButton/TlaCtaButton'
 import { TlaLogo } from '../TlaLogo/TlaLogo'
 import styles from './auth.module.css'
 
 const messages = defineMessages({
 	enterEmailAddress: { defaultMessage: 'Enter your email address' },
+	inviteMessage: {
+		defaultMessage: '<strong>You have been invited to join <groupName></groupName> group</strong>.',
```

Comment:
nit: if the whole thing is strong you could just wrap the `<F>` component below with `<strong>`, or even style it with css.